frontend/src/components/NodeModule.vue

```diff
@@ -1,4 +1,5 @@
 <template>
+  <router-link :to="'/system/' + identifier" tag="v-card">
```

Comment:
Not blocking, but `tag` has been deprecated: https://next.router.vuejs.org/guide/migration/#removal-of-event-and-tag-props-in-router-link